package.json

```diff
@@ -11,6 +11,7 @@
   ],
   "dependencies": {
     "@octokit/core": "^3.5.1",
+    "@octokit/graphql": "^5.0.1",
```

Comment:
I think you can install this as a devDependency instead as we're just using the type from it.